tests/testChebyFits.py

```diff
@@ -106,9 +100,12 @@ def testWrite(self):
         self.cheb.calcSegmentLength()
         self.cheb.generateEphemerides(self.cheb.makeAllTimes())
         self.cheb.calcSegments()
-        self.cheb.write('tmpCoeff', 'tmpResids', 'tmpFailed')
-        self.assertTrue(os.path.isfile('tmpCoeff'))
-        self.assertTrue(os.path.isfile('tmpResids'))
+        with getTempFilePath('.txt') as coeff_name:
+            with getTempFilePath('.txt') as resid_name:
+                with getTempFilePath('.txt') as failed_name:
```

Comment:
I'm surprised this works. I thought that `getTempFilePath` comes up with a predictable name based on the test method name, so can you check that all of these are different names? Given that the test doesn't really notice if they are all the same name I don't think you can tell. This worked in your other examples because you had a different suffix each time. I think `.coeff.txt` and `.resid.txt` etc might be the solution here.